### PR TITLE
Remove GRAPHSAGE

### PR DESCRIPTION
**Pull request by miyurud**
GRAPHSAGE has been used for generating node embeddings. However, we have the capability of generating node embedding using StellarGraph. Hence GRAPHSAGE folder is redundant. Hence with this PR we will be removing GRAPHSAGE folder.

### CODE REVIEW

**Review by Ishad-M-I-M — DISMISSED** [review]

**Comment by Ishad-M-I-M** [comment]
https://github.com/miyurud/jasminegraph/blob/master/README.md?plain=1#L29
Line need to be removed

**Comment by miyurud** [comment]
> https://github.com/miyurud/jasminegraph/blob/master/README.md?plain=1#L29 Line need to be removed

Fixed.

File: src/frontend/JasmineGraphFrontEnd.cpp [hunk 1]
```diff
@@ -1195,23 +1193,16 @@ void *frontendservicesesion(std::string masterIP, int connFd, SQLiteDBInterface
...
-                if (Utils::getJasmineGraphProperty("org.jasminegraph.fl.org.training") == "true") {
-                    frontend_logger.log("Initiate org communication", "info");
-                    jasmineServer->initiateOrgCommunication(graphID, trainData, sqlite);
-
-                } else {
-                    frontend_logger.log("Initiate communication", "info");
-                    jasmineServer->initiateCommunication(graphID, trainData, sqlite);
-                }
+            JasmineGraphServer *jasmineServer = new JasmineGraphServer();
+            if (utils.getJasmineGraphProperty("org.jasminegraph.fl.org.training") == "true") {
```
Comment (thevindu-w) [inline src/frontend/JasmineGraphFrontEnd.cpp:1197]:
```suggestion
            if (Utils::getJasmineGraphProperty("org.jasminegraph.fl.org.training") == "true") {
```

This is a static method.

Comment (miyurud) [reply src/frontend/JasmineGraphFrontEnd.cpp]: Ok, I fixed that in https://github.com/miyurud/jasminegraph/pull/201/commits/ef51224ec5ab0f48167bbbfbb0902fb7af62a5a4

**Review by thevindu-w — APPROVED** [review]
Reviewed

**Pull request merged**